<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="absenButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>590</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Absen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="photoButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>590</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Take Photo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trainButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>590</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Train</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="openCv">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <widget class="QWidget" name="videoApp"/>
    <widget class="QWidget" name="pictureApp"/>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>550</y>
      <width>541</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputNikLine">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>510</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="inputNikButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>500</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="nikLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>510</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NIK</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>openCv</zorder>
   <zorder>absenButton</zorder>
   <zorder>photoButton</zorder>
   <zorder>trainButton</zorder>
   <zorder>statusLabel</zorder>
   <zorder>inputNikLine</zorder>
   <zorder>inputNikButton</zorder>
   <zorder>nikLabel</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
